src/Symfony/Component/Cache/Traits/RelayClusterProxy.php

```diff
@@ -0,0 +1,1203 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Traits;
+
+use Relay\Cluster;
+use Relay\Relay;
+use Symfony\Component\VarExporter\LazyObjectInterface;
+use Symfony\Contracts\Service\ResetInterface;
+
+// Help opcache.preload discover always-needed symbols
+class_exists(\Symfony\Component\VarExporter\Internal\Hydrator::class);
+class_exists(\Symfony\Component\VarExporter\Internal\LazyObjectRegistry::class);
+class_exists(\Symfony\Component\VarExporter\Internal\LazyObjectState::class);
+
+/**
+ * @internal
+ */
+class RelayClusterProxy extends \Relay\Cluster implements ResetInterface, LazyObjectInterface
```

Comment:
As this is a class and not a trait, I suppose this file should be moved outside of the `Traits` sub-namespace?

Reply:
Happy to move it, but there are multiple such classes in namespace `Symfony\Component\Cache\Traits`:
![image](https://github.com/user-attachments/assets/1ceeee81-ce17-4cb9-841a-381c3367db9b)

Should I move all of them? If so, should I move them to `Symfony\Component\Cache\Adapter` or should there be a new namespace, maybe `Symfony\Component\Cache\Adapter\Proxy` or `Symfony\Component\Cache\Proxy`?